### PR TITLE
Add SBLYR to registry

### PR DESCRIPTION
**Pull request by keyflight**

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -1997,5 +1997,20 @@
...
+    "poolId": "8264de3cdb1798dd8758e24cda5101184b44543e7c4421c7815f9ed8",
+    "social": {
+      "twitter": "https://twitter.com/sublayerio",
+      "facebook": "",
+      "reddit": "https://reddit.com/user/sublayerio",
+      "instagram": "",
+      "website": "https://pool.sublayer.io",
+      "youtube": "https://www.youtube.com/channel/UC5MjFaqqlAbTX5NtxbG2_6Q"
+    },
+    "telegram": "sblyr",
```
Comment (lophre) [inline registry.json:2012]: You need to provide a Telegram username.  A group name is not accepted.  The SPA uses Telegram to gather votes from it's members and we use the username to let our members in the voting group.

File: registry.json [hunk 2]
```diff
@@ -1997,5 +1997,20 @@
...
+    "operatorName": "sublayerio",
```
Comment (lophre) [inline registry.json:2013]: Leave this field empty if you don't provide a real first name or full name.

**Review by lophre — CHANGES_REQUESTED** [review]
Please see comments.

**Comment by keyflight** [comment]
@lophre thanks for reviewing. i've adjusted the registry.json, i've also tried pulling the latest registry.json and added my pool in there so you would perhaps be able to merge it at once.

File: registry.json (hunk 2 above)
Comment (keyflight) [reply registry.json]: firstname is Olivier

File: registry.json (hunk 1 above)
Comment (keyflight) [reply registry.json]: personal telegram handle is sublayerio

**Comment by leo42** [comment]
Hey SBLYR, 

Please close your pool request and create from clean.

**Comment by keyflight** [comment]
@leo42 no offence, there's quite some room for improvement in this process. I've first created a PR 3 weeks ago.

**Comment by keyflight** [comment]
@leo42 i prepared it for you; https://github.com/SinglePoolAlliance/Registration/pull/242

**Pull request closed without merging**